setup.cfg

```diff
@@ -1,5 +1,9 @@
 [pytest]
-addopts = -v --cov zdict --cov-report html --cov-report term
+flake8-ignore =
+    zdict/__init__.py ALL
+    zdict/dictionaries/template.py ALL
+
+addopts = --flake8 -v --cov zdict --cov-report html --cov-report term
```

Comment:
Accutually, travis will be affected by `setup.cfg`.

We invoke `setup.py test` in travis, so this config will pollute travis.